### DISCUSSION TITLE
Vue Integration

### DISCUSSION THREAD

**Discussion by AGgup8** [tinyplex/tinybase#147] [General]
Are their any plans for Vue UI integration and composables support?

**Comment by KazimirPodolski** [comment]
It's weird how TinyBase has no presence of Vue at all. I wonder why, is TinyBase somehow just doesn't make sense in a Vue app?

**Reply by jcs224** [reply]
Definitely not the case, I use TinyBase heavily in Nuxt (Vue metaframework). Just don't use any of the fancy React stuff.

**Reply by KazimirPodolski** [reply]
I'm not a Nuxt user but I guess it covers Vue reactivity for TinyBase stuff for you somehow? Or you have your own wrappers?

**Reply by jcs224** [reply]
Yeah I have to write my own wrappers, either in straight-up in Vue components or with the Pinia state manager to have reactive updates on my pages. To be honest, kind of prefer it that way, since I can update Vue/Nuxt and Tinybase independent of one another without relying on Tinybase to keep up with Vue's new features. But I might be unique in that regard
